.travis.yml

```diff
@@ -13,7 +11,4 @@ before_install:
 
 script:
   - ./vendor/bin/phpunit
-
-matrix:
-  allow_failures:
-    - php: 7.0
+  
```

Comment:
now the line contains white characters, and github still shows "no newline at end of file" ...